greenback-kit-core/src/main/java/com/greenback/kit/client/impl/AbstractGreenbackClient.java

```diff
@@ -751,33 +751,33 @@ abstract protected AutoExport deleteAutoExportByUrl(
     //
 
     @Override
-    public AutoExportRun createAutoExportRun(String autoExportId, AutoExportRun autoExportRun) throws IOException {
-        Objects.requireNonNull(autoExportRun, "autoExportRun was null");
+    public ExportRun createExportRun(String autoExportId, ExportRun exportRun) throws IOException {
+        Objects.requireNonNull(exportRun, "ExportRun was null");
 
         final String url = this.buildBaseUrl()
             .path("v2/auto_exports")
             .rel(autoExportId, "runs")
             .toString();
 
-        return this.postAutoExportRunByUrl(url, autoExportRun);
+        return this.postExportRunByUrl(url, exportRun);
     }
 
     @Override
-    public AutoExportRun getAutoExportRunById(String autoExportRunId, Iterable<String> expands) throws IOException {
-        Objects.requireNonNull(autoExportRunId, "autoExportRunId was null");
+    public ExportRun getExportRunById(String exportRunId, Iterable<String> expands) throws IOException {
+        Objects.requireNonNull(exportRunId, "exportRunId was null");
 
         //TODO JB: need to implement this route in lens
         final String url = this.buildBaseUrl()
```

Comment:
Is this true as an endpoint?  Didn't we make this /export_runs/<id> as a path in lens?

Reply:
yeah i think this an old one that Jared put there when we first made it and it has not been updated. I can update it